vault/base/statefulsets/statefulset.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app.kubernetes.io/instance: nerc
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: vault
+  name: vault
+  namespace: vault
+spec:
+  serviceName: vault-internal
+  podManagementPolicy: Parallel
+  replicas: 3
+  updateStrategy:
+    type: OnDelete
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: vault
+      app.kubernetes.io/instance: nerc
+      component: server
+  template:
+    metadata:
+      labels:
+        helm.sh/chart: vault-0.19.0
+        app.kubernetes.io/name: vault
+        app.kubernetes.io/instance: nerc
+        component: server
+    spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchLabels:
+                  app.kubernetes.io/name: vault
+                  app.kubernetes.io/instance: "nerc"
+                  component: server
+              topologyKey: kubernetes.io/hostname
+      terminationGracePeriodSeconds: 10
+      serviceAccountName: vault
+      volumes:
+        - name: config
+          configMap:
+            name: vault-config
+        - name: home
+          emptyDir: {}
+      containers:
+        - name: vault
+          image: quay.io/operate-first/vault:1.9.2
```

Comment:
Is there a registry we can use to store this image related to NERC?

Reply:
@dystewart Tell me more; what is this image for? Why do we need a custom image rather than the upstream `hashicorp/vault:1.10.3` image?

Reply:
@larsks I'm asking because we had to pull the image from Docker and upload it to quay because of docker pull rate issues, which were preventing the image from being pulled successfully on deployment in OperateFirst. With Quay we don't have this rate limiting.  
It's a bit behind the current version bc we have yet to update it in OPF. I'll look into what has been added/changed in the newer versions though. It may make more sense to use the newer version, given I'm able to quickly get it up and confirm things work as currently constituted